contributing/eclipse-google-style.xml

```diff
@@ -65,7 +65,7 @@
 <setting id="org.eclipse.jdt.core.formatter.comment.format_line_comments" value="true"/>
 <setting id="org.eclipse.jdt.core.formatter.insert_space_after_closing_angle_bracket_in_type_arguments" value="do not insert"/>
 <setting id="org.eclipse.jdt.core.formatter.insert_space_after_comma_in_enum_declarations" value="insert"/>
-<setting id="org.eclipse.jdt.core.formatter.join_wrapped_lines" value="false"/>
```

Comment:
@csviri @metacosm this is the only change I had to do in order for spotless to not mess things up. But it unrolled a lot of lines.

Reply:
I would rather stay on an older version while is this fixed in spotless, or is it not expected to be fixed?